analytics/terraform/emr-eks-karpenter/helm-values/kube-prometheus-amp-enable.yaml

```diff
@@ -5,6 +5,8 @@ prometheus:
     annotations:
       eks.amazonaws.com/role-arn: ${amp_irsa}
   prometheusSpec:
```

Comment:
Could you update all the changes applied to `prometheusSpec` to `helm-values`kube-prometheus.yaml` file as well. This will help users who are not using AMP setup by default